src/blocks/remote-data-html/edit.tsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	transformStyles,
+	store as blockEditorStore,
+	EditorStyle,
+	BlockEditorStoreSelectors,
+	useBlockProps,
+} from '@wordpress/block-editor';
+import { BlockEditProps } from '@wordpress/blocks';
+import { SandBox, Placeholder } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { useEffect, useMemo } from '@wordpress/element';
+
+import { __ } from '@/utils/i18n';
+
+import './editor.scss';
+
+/**
+ * This block is based on the official core/html block from Gutenberg.
+ * The main difference is that this block binds to HTML from remote data block context instead of user input.
+ */
+
+// Default styles used to unset some of the styles
+// that might be inherited from the editor style.
+const DEFAULT_STYLES = `
+	html,body,:root {
+		margin: 0 !important;
+		padding: 0 !important;
+		overflow: visible !important;
+		min-height: auto !important;
+	}
+`;
+
+export default function RemoteDataHTML(
+	props: BlockEditProps< RemoteDataInnerBlockAttributes >
+): JSX.Element {
+	const { attributes, setAttributes, isSelected } = props;
+	const blockProps = useBlockProps();
+
+	// Pass the content provided by bindings to save() via setAttributes with changed content
+	const { content } = attributes;
+	useEffect( () => {
+		setAttributes( { content } );
+	}, [ content, setAttributes ] );
+
+	const settingStyles = useSelect< BlockEditorStoreSelectors, EditorStyle[] >(
+		select => select( blockEditorStore ).getSettings().styles,
+		[]
+	);
+
+	const styles = useMemo(
+		() => [
+			DEFAULT_STYLES,
+			...transformStyles( ( settingStyles ?? [] ).filter( ( style: EditorStyle ) => style.css ) ),
+		],
+		[ settingStyles ]
+	);
```

Comment:
`transformStyles` is internally cached, don't need memo

```suggestion
	const styles = [
		DEFAULT_STYLES,
		...transformStyles( ( settingStyles ?? [] ).filter( ( style: EditorStyle ) => style.css ) ),
	];
```